api_docs/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 284**
+
+* [`GET /events`](/api/get-events), [`GET /messages`](/api/get-messages),
+  [`GET /messages/{message_id}`](/api/get-message),
+  [`POST /zulip-outgoing-webhook`](/api/zulip-outgoing-webhooks): Removed
+  the `prev_rendered_content_version` field from the `edit_history` object
+  within message objects and the `update_message` event type as it is not
+  used by any client.
```

Comment:
I would "s/as it is not used .../as it is an internal server implementation detail not used by any client/", just borrowing a bit of language from the event documentation being deleted.

Reply:
<img width="703" alt="Screenshot 2024-08-29 at 12 56 54 PM" src="https://github.com/user-attachments/assets/9b19afaf-abb0-40c4-8c09-1d4f9398d902">